fastify.d.ts

```diff
@@ -68,8 +80,16 @@ type FastifyHttpsOptions<
 > = FastifyServerOptions<Server, Logger> & {
   https: https.ServerOptions
 }
+
+type FastifyHttpOptions<
```

Comment:
Can you add a comment for this one.

Reply:
Sure I'll put up a new pull request tomorrow. This is just the options taken by the constructor when the server is an insecure HTTP server. It's by all means an alias of the ServerOptions.

Reply:
Yes exactly, thanks!